azure/v12.0.1/README.md

```diff
@@ -0,0 +1,20 @@
+## :zap:  Giant Swarm Release 12.0.1 for Azure :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades external-dns app to v0.7.3.
+
+**Important Warning**
```

Comment:
@sslavic and maybe @giantswarm/team-celestial , is the Important Warning section a duplicate of Notes for future 12.x.x release? I fear it is two warnings about the same thing? I could be wrong.

Reply:
I copied this stuff from 12.0.0 release notes

They do overlap to some extent, related to the warning that during upgrade from 11.4.0 to 12.0.x there could be requests failure (downtime) observed. Although they express it a bit differently - maybe rationale for the two variants of the message with different level of details is that one is meant for customer while second one for us publishing new releases.

There's also some uniqueness, e.g. lower one has warning that to upgrade to 12.x one has to upgrade to 11.4.0 first.

We can keep just lower one. But I'm fine in dropping both too.

Reply:
Kept just "Note for future 12.x.x release", PTAL